src/test/java/ru/yandex/practicum/filmorate/FilmorateApplicationTests.java

```diff
@@ -2,12 +2,91 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.model.User;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.junit.jupiter.api.BeforeAll;
+import java.time.LocalDate;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @SpringBootTest
 class FilmorateApplicationTests {
 
+	private static Validator validator;
+
+	@BeforeAll
+	static void setUp() {
+		ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+		validator = factory.getValidator();
+	}
+
+	@Test
+	public void testBadEmail() {
+		User user = new User("badEmail",
+				"ErevosX", "Alex", LocalDate.of(2010, 9, 7));
+		Set<jakarta.validation.ConstraintViolation<User>> violations = validator.validate(user);
```

Comment:
Нужно добавить в импотры `import jakarta.validation.ConstraintViolation;` и убрать полный путь до объекта здесь и ниже 
```suggestion
		Set<ConstraintViolation<User>> violations = validator.validate(user);
```